doc/Programming-guide/Your-first-Kaa-application/index.md

```diff
@@ -2,7 +2,615 @@
+1. In the Add application window, enter the application name and then click **Add**.
+For this guide we will use **Trustful** credential service and our application will be
+named "My First Kaa Application"
+
+    ![Admin Console](attach/new_app.png)
+
+After the application has been added, you may log out. We will not be using
+the tenant administrator role in this guide anymore.
+
+# Creating schemas
+
+The application that you have created in the previous step already includes
+the default versions of the profile, configuration, notification and log schemes
+ready for use. However, in this sample application, we will use a custom data
+collection and configuration schemas for demonstration purposes. Default value for
+sampling period is set to 1 second.
+
+To create and upload the schemas, proceed as follows:
+
+1. Create the *data-schema.json* file on your PC with the following schema definition:
+
+        {
+            "type": "record",
+            "name": "DataCollection",
+            "namespace": "org.kaaproject.kaa.schema.example",
+            "fields": [
+                {
+                    "name": "temperature",
+                    "type": "int"
+                }
+            ]
+        }
+1. Create the *configuration-schema.json* file and add following schema:
+
+        {
+            "type": "record",
+            "name": "Configuration",
+            "namespace": "org.kaaproject.kaa.schema.example",
+            "fields": [
+                {
+                    "name": "samplePeriod",
+                    "type": "int",
+                    "by_default": "1"
+                }
+            ]
+        }
+1. Open the admin UI in your browser and log in as
+    a tenant developer (user/password: devuser/devuser123).
+1. Open the relevant **Log schemas** window (**Applications =>
+    My First Kaa Application => Schemas => Log**) and click **Add schema**.
+1. In the **Add log schema** window, enter the name and description
+    of the new data collection schema.
+
+    ![Data collection scheme](attach/new_data_schema.png)
+1. Scroll down and use the **Upload from file** function to find the previously
+    created json file with the schema. Alternatively, you can use the
+    [Schema Avro UI]({{root_url}}/Administration-guide/Tenants-and-applications-management/#avro-ui-forms).
+    form to create the schema.
+1. Click **Upload**.
+1. Click **Add** at the top of the window.
+1. Repeat uploading and adding for the configuration schema
+    (*configuration-schema.json*) scheme:
+
+    ![Data collection scheme](attach/new_config_schema.png)
+
+As a result of these operations you will see configuration and data collections
+schemes in the list:
+![Data collection schema complete](attach/log_schema_list.png)
+![Configuraion schema complete](attach/config_schema_list.png)
+
+In this screenshot, version 2.0 is log and configuraion schema version that
+was just created. We will use this version for the SDK generation later.
```

Comment:
There is no mention of '2.0' on the screen-shot.
This either needs an additional explanation or just using of '2~~.0~~'.